DESCRIPTION

```diff
@@ -73,3 +73,18 @@ Collate:
     'importPackages.R'
     'plot.R'
 VignetteBuilder: knitr
+
+Config/reticulate:
+  list(
+    packages = list(
+      list(package = "umap"),
+      list(package = 'numpy'),
+      list(package = "scipy"),
+      list(package = "torch"),
+      list(package = "umap-learn")
+    )
+  )
```

Comment:
I tried it and it tries to install all the packages on my system. Actually got an error for scipy but I am not sure if its my setup. Afterwards it worked.